process_manager/templates/process_manager/index.html

```diff
@@ -11,6 +11,10 @@
       max-height: 80vh;
       overflow-y: auto;
     }
+    #hide-messages-button {
+      /* Certain themes invert the colour, so we invert back. */
+      filter: invert(1);
```

Comment:
You can add this line above since you are already editing the attribute above.
`#hide-messages-button {
      float: right;
      filter: invert(1);
}`

Reply:
Oh yes, not paying attention. Thanks!